DfE.FindInformationAcademiesTrusts/Setup/ConfigurationVariables.cs

```diff
@@ -12,6 +13,25 @@ public static void BindConfigurationVariables(WebApplicationBuilder builder)
     {
         if (builder.Environment.IsLocalDevelopment())
             builder.Configuration.AddUserSecrets(Assembly.GetExecutingAssembly());
+        
+        // Retrieve the connection string
+        string? appConfigConnectionString = builder.Configuration.GetConnectionString("AppConfig");
+
+        // Load App Configuration and Feature Flags from Azure
+        if (!string.IsNullOrEmpty(appConfigConnectionString))
```

Comment:
Do we expect this to be present in certain environments? If so then it would be good to know if the `appConfigConnectionString` was empty/missing so we know that the configuration is incorrectly set up

Reply:
The azure app configuration isn't actually set up in the envs at the moment which is why we can't switch on env. For now we're going to log a warning if the config is missing (so we don't silently miss it) and then revisit when the resources are actually configured